### PR TITLE
Docker image tweaks

### PR DESCRIPTION
**Pull request by mjurbanski-reef**

### CODE REVIEW

File: Dockerfile.template [hunk 1]
```diff
@@ -11,8 +11,8 @@ LABEL build-date-iso8601="${build_date}"
 
 WORKDIR /root
 
-COPY ${tar_path}/${tar_name} .
```
Comment (mjurbanski-reef) [inline Dockerfile.template:14]: ~170kB; but perhaps more imortant it looked weird to just have that tar left in there

Comment (mjurbanski-reef) [inline Dockerfile.template:14]: ~170kB; but perhaps more imortant it looked weird to just have that tar left in there

Comment (mjurbanski-reef) [inline Dockerfile.template:14]: ~170kB; but perhaps more imortant it looked weird to just have that tar left in there

Comment (mjurbanski-reef) [inline Dockerfile.template:14]: ~170kB; but perhaps more imortant it looked weird to just have that tar left in there

Comment (mjurbanski-reef) [inline Dockerfile.template:14]: ~170kB; but perhaps more imortant it looked weird to just have that tar left in there

File: Dockerfile.template [hunk 2]
```diff
@@ -11,8 +11,8 @@ LABEL build-date-iso8601="${build_date}"
...
-RUN ["pip", "install", "${tar_name}[full]"]
```
Comment (mjurbanski-reef) [inline Dockerfile.template:15]: --no-cache-dir saves ~6MB

File: Dockerfile.template [hunk 3]
```diff
@@ -1,4 +1,4 @@
-FROM python:${python_version}-slim as base
+FROM python:${python_version}-alpine as base
```
Comment (mjurbanski-reef) [inline Dockerfile.template:1]: suprisingly enough, alpine switch "just works"; seems all binary dependencies do have musl wheel

Comment (mjurbanski-reef) [inline Dockerfile.template:1]: suprisingly enough, alpine switch "just works"; seems all binary dependencies do have musl wheel

Comment (mjurbanski-reef) [inline Dockerfile.template:1]: using `builder` stage did not help save any disk space and was troublesome to get working - you have to be really careful what you want to copy, which makes Dockerfile significantly more complex.

We probably could do our custom python-base imagine, but I don't see that paying off as we don't even know how many MBs we can save that way.
If we ever start using `standalone` binary in these images instead, we are likely to see better results than optimizing python install.

Comment (mpnowacki-reef) [reply Dockerfile.template:1]: ok but the `builder` stage wasn't there when you started working on the thing, right?

**Review by mpnowacki-reef — APPROVED** [review]

File: Dockerfile.template (hunk 3 above)
Comment (mjurbanski-reef) [reply Dockerfile.template:1]: there was no builder stage

**Comment by mjurbanski-reef** [comment]
closed as alpine may degrade performance (unconfirmed)

**Pull request closed without merging**